src/main/java/sirius/kernel/Sirius.java

```diff
@@ -287,10 +299,12 @@ private static void setupLocalConfig() {
             // instance.conf and develop.conf are not used to tests to permit uniform behaviour on local
             // machines and build servers...
             if (Sirius.isDev()) {
-                config = setup.applyDeveloperConfig(config);
+                config = setup.applyConfig(config, CONFIG_DEVELOP);
+            } else if (Sirius.isStaging()) {
+                config = setup.applyConfig(config, CONFIG_STAGING);
```

Comment:
didn't we also want to add a prod.conf? to configure production only settings?

Reply:
Isn't that effectively our instance.conf?

Reply:
I got it wrong, first. I now have added a productive.conf check.